src/atom-environment.coffee

```diff
@@ -117,7 +117,7 @@ class AtomEnvironment extends Model
 
   # Call .loadOrCreate instead
   constructor: (params={}) ->
-    {@blobStore, @applicationDelegate, @window, @document, configDirPath, @enablePersistence} = params
+    {@blobStore, @applicationDelegate, @window, @document, configDirPath, @enablePersistence, loadBaseStylesheetsOnly} = params
```

Comment:
:art: Any ideas on how to call this parameter? I am not super happy with `loadBaseStylesheetsOnly` but it's probably okay.